src/routes/sentry-example/+page.svelte

```diff
@@ -0,0 +1,97 @@
+<!-- 
+This is just a very simple page with a button to throw an example error.
+Feel free to delete this file and the entire sentry route.  
```

Comment:
If we plan to keep it, we can remove this line of the comment

Reply:
just wanna keep it for testing stage 👍🏻